sidebars.js

```diff
@@ -776,6 +776,16 @@ module.exports = {
         "apis-tools/build-your-own-client",
       ],
     },
+    {
+      SDKs: [
+        {
+          "Spring Zeebe": [
```

Comment:
Is this the name we want to use? I've interchangeably seen Spring Boot Starter Camunda SDK.

Reply:
Yes